modules/apps/dynamic-data-mapping/dynamic-data-mapping-test-util/src/main/java/com/liferay/dynamic/data/mapping/test/util/DDMFormTestUtil.java

```diff
@@ -244,6 +256,15 @@ public static DDMFormField createLocalizableTextDDMFormField(String name) {
 		return createTextDDMFormField(name, true, false, false);
 	}
 
+	public static DDMFormField createLocalizedTextDDMFormField(
+		String name, boolean localizable, boolean repeatable, boolean required,
```

Comment:
@G-Sa if this method is `createLocalizedTextDDMFormField` then we don't need the `boolean localizable`